### DISCUSSION TITLE
FIRELY_III_LAYOUT=v2 not working =?

### DISCUSSION THREAD

**Discussion by ghost** [firefly-iii/firefly-iii#7925] [Firefly III] [answered]
Hey,

I just saw the new release and wanted to give the beta layout a chance.
But the layout is still the same or am I missing something?

I added `FIRELY_III_LAYOUT=v2` to `.env`.

Thanks i advantage!

**Comment by JC5** [comment, marked as answer]
The spelling in the change log is wrong. It should be fireFly.

**Reply by ghost** [reply]
er.. thanks. haven't seen that one
